backend/UniTodo/.editorconfig

```diff
@@ -2,3 +2,85 @@
 
 # IDE0058: Expression value is never used
 csharp_style_unused_value_expression_statement_preference = unused_local_variable
+csharp_using_directive_placement = outside_namespace:silent

```

Comment:
can you have this file on a separate commit with a description on why these changes are needed?

Reply:
I found out that this is a file Visual Studio adds to enforce specific styling on the editor. So perhaps this may not break anything, but it also doesn't need to be merged? I could revert this change

Reply:
Feel free to leave it here. Just add it on a separate commit ;) So we have a cleaner history